Makefile.am

```diff
@@ -30,7 +30,7 @@ if ENABLE_EXAMPLES
 SUBDIRS += third-party examples
 endif
 
-dist_doc_DATA = README.rst
+dist_doc_DATA = README.rst SECURITY.md
```

Comment:
This installs SECURITY.md under $prefix/share/doc/ngtcp2.  If this file is just needed in an archive, append it to EXTRA_DIST.